imports/plugins/included/sitemap-generator/client/components/SitemapSettings.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/styles";
+import { Components } from "@reactioncommerce/reaction-components";
+import Select from "@reactioncommerce/components/Select/v1";
+import { useMutation, useQuery } from "@apollo/react-hooks";
+import Button from "@reactioncommerce/catalyst/Button";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import FormLabel from "@material-ui/core/FormLabel";
+import Grid from "@material-ui/core/Grid";
+import { i18next } from "/client/api";
+import Logger from "/client/modules/logger";
+import generateSitemapsMutation from "../mutations/generateSitemaps";
+import updateShopSettingsMutation from "../mutations/updateShopSettings";
+import shopSettingsQuery from "../queries/shopSettings";
+
+const useStyles = makeStyles(() => ({
+  card: {
+    // Without this, the Select dropdown menu gets cut off at the bottom of the card
+    overflow: "visible"
```

Comment:
@catalystpod (@mikemurray @machikoyasuda @willopez @rymorgan @cassytaylor)

See @aldeed's comment here. We've noticed this getting cut off in a few places, the `Origin country` of the Product admin comes to mind as one of them.

Reply:
Also, I first tried using the Catalyst Select, but that one does not accept a `value` prop with the current value so it's kind of useless for an update form such as this. I will submit a catalyst issue about that when I get a chance.